static/sass/patterns/_sections.scss

```diff
@@ -1,31 +1,26 @@
 // Sections Pattern
 // ================
 
+
 [data-module] {
-  border-bottom: 2px dotted color('action');
-  padding-bottom: 2rem;
+  margin: size('gutter') auto size('spacer');
+  max-width: size('full-content');
+  padding: 0 size('gutter');
 
   & + & {
-    padding-top: 2rem;
+    @include before(' • • • ') {
```

Comment:
these separators are nice, but it feels like something is truncated since it is a large ellipses. Wonder if we should add something on each side, or make it 5 dots or something?